src/app/(proper_react)/(onboarding)/user/welcome/View.tsx

```diff
@@ -0,0 +1,120 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { useState } from "react";
+import Link from "next/link";
+import Image from "next/image";
+import { Session } from "next-auth";
+import styles from "./View.module.scss";
+import monitorLogo from "../../../images/monitor-logo.webp";
+import stepGetStartedIcon from "./images/step-counter-get-started.svg";
+import stepEnterInfoIcon from "./images/step-counter-enter-info.svg";
+import stepFindExposuresIcon from "./images/step-counter-find-exposures.svg";
+import stepDoneIcon from "./images/step-counter-done.svg";
+import { useL10n } from "../../../../hooks/l10n";
+import { GetStarted } from "./GetStarted";
+import { FindExposures } from "./FindExposures";
+import { EnterInfo } from "./EnterInfo";
+
+export type Props = {
+  user: Session["user"];
+};
+
+type StepId = "getStarted" | "enterInfo" | "findExposures";
+
+export const View = (props: Props) => {
+  const l10n = useL10n();
+  const [currentStep, setCurrentStep] = useState<StepId>("getStarted");
```

Comment:
Note that I'm storing the user's progress through the step in-memory client-side. This made sense to me: if you refresh before you finished entering your info, it's no biggie to end up at the welcome screen again. If you refresh after entering your info (i.e. while the progress bar is loading), you'll end up at the dashboard.

Reply:
In PR https://github.com/mozilla/blurts-server/pull/3157 I saw that @rhelmer has added dedicated routes for each step, which I’m not opposed to. If there is no real benefit to why we would need to link a user to a step of the flow this approach will work fine as well. Otherwise, we could still use query params right?

Reply:
I don't think separate routes make much sense - what would it mean for a user to skip right ahead to step three without having entered their details? This is also why I didn't make the step indicator elements clickable. I was thinking more about potentially keeping track of the user's progress through the flow on the server-side.

But yes, it would be easier to transition from this to separate URLs (either via the path or via query params) down the road, than it would be the other way around, as that would risk breaking links.